chanzuckerberg/idseq-web
# Hook coverage viz up to back-end endpoints.

Remove mock data from front-end.

Back-end uses mock data from S3.
This mock data is the coverage viz pipeline step run on our standard benchmark sample.
We will remove the mock data and change the back-end to point to the correct files when we add the coverage-viz pipeline step to idseq-web.

--
Testing steps:
Verified that non-admin users can still open old alignment viz without error.
If admin user goes to benchmark sample run, they can see coverage viz with data.
If admin user goes to non-benchmark sample, they can see coverage viz with data if the taxon they clicked on is also included in the benchmark sample (as expected). Otherwise, they see the 'no-data' message without error.


![Screen Shot 2019-04-17 at 1 35 23 PM](https://user-images.githubusercontent.com/837004/56319581-f0e19080-6116-11e9-96a2-dd159237643d.png)
